### ISSUE TITLE
Gremlin identifier mangling (cleanOrientId) breaks subsequent queries for namespaced ids

### ISSUE THREAD

**Issue by rbramley** [ArcadeAnalytics/arcade-connectors#97]
**Describe the bug**
A namespaced identifier, e.g. `acme:12` is _cleaned_ to replace colons with underscores (e.g. `acme_12`). This is not correctly handled in the reverse direction, causing queries that use identifiers to return 0 results due to identifier mismatch.

**To Reproduce**
Steps to reproduce the behaviour:

0. Create a simple graph in Gremlin:
```
    g.addV('product').property(id, 'acme:12').property('name', 'ACME TNT')
    g.addV('character').property(id, 'cartoon:wc').property('name', 'Wily Coyote')
    g.addE('USES').from(g.V('cartoon:wc')).to(g.V('acme:12'))
```
1. In Arcade Analytics, create a Gremlin datasource (I'm using Neptune) and index it
2. Add a new widget
3. Select a graph entity by class, using '_product_'
4. 1 node will be rendered
5. Select the node and attempt to traverse the inbound '_USES_' edge
6. The arcadeanalytics app console will show a logged query e.g. 
`c.a.p.gremlin.GremlinDataProvider        : fetching data from 'xyz.neptune.amazonaws.com/10.nn.nn.nn:8182' with query 'g.V( 'acme_12' ).inE("USES")' `
and the resulting log entry: 
`c.a.p.gremlin.GremlinDataProvider        : fetched 0 nodes and 0 edges `
7. The user interface therefore cannot be updated to show the incoming edge and source node.

**Expected behaviour**
The expectation is that (_doubly_) namespaced identifiers (e.g. _ARCADE_ID_ `1301_acme_12`) are correctly unmangled during query construction so that correct queries are issued to the Gremlin endpoint. e.g. `'g.V( 'acme:12' ).inE("USES")'`

**Comment by rbramley** [comment]
The problem is in https://github.com/ArcadeAnalytics/arcade-connectors/blob/master/gremlin/src/main/kotlin/com/arcadeanalytics/provider/gremlin/GremlinDataProvider.kt#L241 as it does not have a `GREMLIN_NEPTUNE` condition to `id.replace("_", ":")`

**Comment by arcade-player** [comment]
The code is fixed, but not released. As you can see in the code, the replacement fo : with _ is done only when the DataSource is OrientDB_gremlin. We will relese 1.0.8 in a couple of days.

**Comment by arcade-player** [comment]
released  and arcadeanalytics 1.0.3 that includes arcade-connectors 1.0.8, that should resolve the problem. 
Le me know.
